lib/themes/dosomething/paraneue_dosomething/templates/system/page--full-width.tpl.php

```diff
@@ -12,6 +12,7 @@
 <?php if (!empty($tabs['#primary'])): ?><nav class="admin-tabs"><?php print render($tabs); ?></nav><?php endif; ?>
 <?php print $messages; ?>
 
+<div id="onboarding" class="wrapper"></div>
```

Comment:
It might be nice to think about a convention for React component containers to make it clear what this is used for, maybe `#jsx-onboarding` or `#react-onboarding`?